apps/dashboard/src/lib/components/OrgSelector/OrgSelector.svelte

```diff
@@ -37,7 +37,7 @@
       <p class="dark:text-white text-lg font-bold whitespace-nowrap truncate">
         {$currentOrg.name}
       </p>
-      <CaretSortIcon size={16} class="ml-2" />
+      <CaretDown size={16} class="ml-2"/>
```

Comment:
```suggestion
      <CaretDown size={16} class="ml-2" />
```

Reply:
Done with change @rotimi-best

Reply:
Thanks a lot for this.

Can you please share your Twitter handle so I can give you a shout out

Reply:
sure @rotimi-best , My twitter -  https://twitter.com/AnishYadav32381